plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -161,6 +166,7 @@ protected function validate(array $data)
                 ->notEmpty('username');
         } else {
             $validator
+                ->allowEmpty('activation_url')
```

Comment:
Maybe only when `Signup.activationUrl` is empty? 🤔 

```php
$validator
    ->allowEmpty('activation_url', function () {
        return !Configure::check('Signup.activationUrl');
    })
    // ...
```

Reply:
Not really. In this case we are using an external auth provider, so we don't usually need an activation URL.
But we can write assignement better in `execute()`